project/driver_controls/src/event_arbiter.c

```diff
@@ -0,0 +1,50 @@
+#include <stdio.h>
```

Comment:
The lack of an init function makes it pretty difficult to test properly, since you'll run out of storage.